app/index.html

```diff
@@ -1,16 +1,16 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>LibraryApp - Ember Octane Tutorial</title>
-    <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     {{content-for "head"}}
 
-    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/library-app.css">
+    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css" />
+    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/library-app.css" />
```

Comment:
I wonder why `prettier` is adding the closing `/` to `meta` and `link` here. None of the examples shown in the [Living HTML Standard](https://html.spec.whatwg.org/#the-meta-element) use the `/`.

https://html.spec.whatwg.org/#the-meta-element
https://html.spec.whatwg.org/#the-link-element

Reply:
There is some discussion about it here https://github.com/prettier/prettier/issues/5246

The main point of using Prettier, that the most of the formatting decision is made by the Prettier community, so a dev team doesn't have to worry about formatting decision. ;) It looks to me that html is right with or without closing tag.

I prefer to customise prettier rules as minimal as possible. So if Prettier prefer this, I would suggest, let's keep in this way. :)